.github/actions/setup-job/action.yml

```diff
@@ -33,14 +35,14 @@ runs:
 
     - name: Install flutter
       uses: subosito/flutter-action@5e1529bc12adae5b26f30f0a4046ac743d6f0c5d # v2.3.0
-      if: inputs.flutter
+      if: inputs.flutter != 'false'
```

Comment:
As long as `inputs.flutter` can only be `true`/`false` this is not necessary.

Or to avoid future problems we could use `!= 'false'` everywhere `inputs.flutter/dart/rust` is used.

Reply:
with out this it was always installing the flutter stuff, not sure why.

Reply:
i think the problem is that `default` is always a string 
https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault

Reply:
Probably, before for the "false" case I simply had no default specified.

Githubs type handling is so strange, it's like JSON typed, but then requires you to specify thing stringly typed and sometimes it's basically stringly typed. :pouting_cat: 

Like:

>  If the types do not match, GitHub coerces the type to a number. GitHub casts data types to a number using these conversions:

except somewhere below this is hidden:

> Some functions cast values to a string to perform comparisons.

and boolean conversion is not even mentioned...